activityframe.lua

```diff
@@ -256,6 +256,10 @@ function activity.RenderKeyFinishedMarker(frame, event, marker)
         marker.SubFrames.icon = icon
     end
 
+    if (event.arguments.timeLostToDeaths and event.arguments.timeLostToDeaths > 0) then
+        marker.TimestampLabel:SetText(marker.TimestampLabel:GetText() .. " (+" .. detailsFramework:IntegerToTimer(event.arguments.timeLostToDeaths) .. ")")
```

Comment:
'GetText' seems bad. Can that be omitted?

Reply:
Certainly, I've changed it in the latest commit